lib/python/IMOSnetCDF.py

```diff
@@ -175,20 +173,20 @@ def updateAttributes(self):
         # TIME
         if self.variables.has_key('TIME'):
             time = self.variables['TIME']
-            self.time_coverage_start = (epoch + timedelta(min(time[:]))).isoformat() + 'Z'
-            self.time_coverage_end   = (epoch + timedelta(max(time[:]))).isoformat() + 'Z'
+            self.time_coverage_start = (epoch + timedelta(time[:].min())).isoformat() + 'Z'
+            self.time_coverage_end   = (epoch + timedelta(time[:].max())).isoformat() + 'Z'
 
         # LATITUDE
         if self.variables.has_key('LATITUDE'):
             lat = self.variables['LATITUDE']
-            self.geospatial_lat_min = min(lat[:])
-            self.geospatial_lat_max = max(lat[:])
+            self.geospatial_lat_min = lat[:].min()
+            self.geospatial_lat_max = lat[:].max()
 
         # LONGITUDE
         if self.variables.has_key('LONGITUDE'):
             lon = self.variables['LONGITUDE']
-            self.geospatial_lon_min = min(lon[:])
-            self.geospatial_lon_max = max(lon[:])
+            self.geospatial_lon_min = lon[:].min()
```

Comment:
I had an issue lately with this. Some netcdf files have LATITUDE and LONGITUDE set as dimensionless var. And because of this, they have a shape of 0. and min and max can't be performed on 0-d array. So maybe worth handling this type of scenario as well